modules/prov/src/main/resources/schemas/nexus/prov/organization/v0.1.0.json

```diff
@@ -1,14 +1,14 @@
 {
   "@context": [
-    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    "{{base}}/contexts/nexus/core/schema/v0.2.0",
     {
-      "this": "{{base}}/schemas/nexus/prov/organization/v0.1.0/shapes/"
+      "@base": "{{base}}/schemas/nexus/prov/organization/v0.1.0/shapes/"
     }
   ],
   "@type": "nxv:Schema",
   "shapes": [
```

Comment:
should there be an import of the respective provsh schema?

Reply:
In W3C PROV-0, Organization is not strongly constrained so that applications can decide how to describe an Organization.

Reply:
I see! Thanks!